Sources/SwiftGenKit/Parsers/StoryboardParser/StoryboardScene.swift

```diff
@@ -0,0 +1,92 @@
+//
+// SwiftGenKit
+// Copyright (c) 2017 SwiftGen
+// MIT Licence
+//
+
+import Foundation
+import Kanna
+
+func uppercaseFirst(_ string: String) -> String {
+  guard let first = string.first else {
+    return string
+  }
+  return String(first).uppercased() + String(string.dropFirst())
+}
+
+extension Storyboard {
+  struct Scene {
+    let identifier: String
+    let tag: String
+    let customClass: String?
+    let customModule: String?
+    let platform: Platform
+
+    private static let tagTypeMap = [
+      "avPlayerViewController": "AVPlayerViewController",
+      "glkViewController": "GLKViewController",
+      "pagecontroller": "NSPageController"
+    ]
+
+    private static let tagModuleMap = [
+      "avPlayerViewController": "AVKit",
+      "glkViewController": "GLKit"
```

Comment:
Maybe we could find a way to avoid repeating magic strings like this in those two maps. Probably not worth a dedicated enum or constants but maybe with a single map with tuple values? (Could be for separate PR, sorry not to have caught that in the previous one)

Reply:
Will do. This is a "refactor" PR, so might as well do it here 🤷‍♂️